src/features/storycreator/storycreator.slice.ts

```diff
@@ -0,0 +1,279 @@
+import { createSelector, createSlice } from '@reduxjs/toolkit';
+
+import type { RootState } from '@/features/common/store';
+
+
+export interface StoryCreatorState {
+  stories: Array<any>;
```

Comment:
follow-up: might make sense to make this a `Record<Story['id'], Story>`

Reply:
also: do we ever need to access `slides` outside the context of a story? i.e. should `slides` just be a property of the story:

```ts
interface Story {
  id: string
  title: string
  slides: Array<Slide>
}
```